.env.example

```diff
@@ -1,7 +1,10 @@
-DB_USERNAME=postgres
-DB_PASSWORD=db_password
-DB_NAME=barefoot_nomad
-DB_HOST=127.0.0.1
+DB_USERNAME=
+DB_PASSWORD=
+DB_NAME=
+DB_TEST_NAME=
+DB_HOST=
 PORT=
+DB_PORT=
 
-DB_TEST_NAME=barefoot_test
+SALT=
```

Comment:
why this?

Reply:
SALT is an additional data input when hashing a password or number of rounds needed for a password to be hashed